tests/unit/Codeception/Lib/Driver/MysqlTest.php

```diff
@@ -51,6 +51,21 @@ public function tearDown()
         }
     }
 
+    /**
+     * If the method exists (PHPUnit 5) forward the call to the parent class, otherwise
+     * call `expectException` instead (PHPUnit 6)
+     */
+    public function setExpectedException($exception, $message = '', $code = null)
```

Comment:
Ugh, you added this method to 8 test files.
The most elegant solution would be to migrate tests (all?) from PhpUnit_Framework_Testcase to `Codeception\Test\Unit`, but more pragmatic solution would be to add a trait.